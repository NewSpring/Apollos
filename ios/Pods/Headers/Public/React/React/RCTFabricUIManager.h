../../../../../../node_modules/react-native/React/Fabric/RCTFabricUIManager.h