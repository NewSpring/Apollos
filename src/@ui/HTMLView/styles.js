import styled from '@ui/styled';
import { Text } from 'react-native';

const Link = styled(({ theme }) => ({
  color: theme.colors.common.primary,
}), 'HTMLView.Link')(Text);

export { Link };
