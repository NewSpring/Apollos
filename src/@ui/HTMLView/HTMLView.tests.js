import React from 'react';
import renderer from 'react-test-renderer';

import Providers from '@ui/TestProviders';
import HTMLView from './';

describe('the HTMLView component', () => {
  it('renders HTML to React', () => {
    const tree = renderer.create(
      <Providers>
        <HTMLView>
          {'<p>I sat on my couch next to my high school <a href=\'https://newspringfuse.com/groups\' target=\'_blank\'>Fuse Group</a>, silently fuming at their inability to put their phones down during Bible study. Finally, I spoke up, teasing one of the girls in a tone that clearly said, &ldquo;I&rsquo;m not actually kidding.&rdquo;&nbsp;</p>\n\n<p>A few days later, I was in the car with my sister, and as I sat in the passenger seat responding to emails, I heard her laugh and say, &ldquo;You always mumble when you are typing. Stop doing that.&rdquo;&nbsp;</p>\n\n<p>&ldquo;Sorry,&rdquo; I responded. &ldquo;I just don&rsquo;t want people to think I&rsquo;m ignoring them when I&rsquo;m typing.&rdquo;</p>\n\n<p>But the truth is, I am ignoring them! In my head, this behavior is totally acceptable. I tell myself I&rsquo;m tending to my priorities in order, and in doing so, I send a message to the person in front of me that she is of lesser importance.</p>\n\n<blockquote>\n<p>My phone is the primary outlet that distracts me from people who need my attention.</p>\n</blockquote>\n\n<p>I quickly get frustrated when I am the one on the outside, but how often to I shut others out during my own social media binges or email frenzies? Just like my girls, I&rsquo;m learning that my phone is the primary outlet that distracts me from people who need my attention.</p>\n\n<h3>Why Being Present Matters</h3>\n\n<p>There is nothing wrong with using our phones &mdash; as long as we remember that the best investment we can make is to invest in our relationships with Jesus and each other. We were created for <a href=\'http://newspring.cc/community\' target=\'_blank\'>community</a> by God, who exists in community as Father, Son, and Holy Spirit (John 1:1-2, Genesis 2:18).&nbsp;</p>\n\n<p>Our phones help us stay connected, but they are not the foundation of all connection. If we&rsquo;re honest, our phones are much quicker to distract and disengage us from the people right in front of us.&nbsp;</p>\n\n<p>Jesus said loving God and loving others is greatest of all God&rsquo;s commandments (Matthew 22:38-40). And, after trying everything the world had to offer, King Solomon reminded his people that their relationships with God and others are the only things that really matter (Ecclesiastes 2:24-26, Ecclesiastes 3:9).&nbsp;</p>\n\n<blockquote>\n<p>Greater love has no one than this: to lay down one&rsquo;s phone for one&rsquo;s loved ones.</p>\n</blockquote>\n\n<p>So the next time we feel like we have to text back immediately or tweet before someone else does, consider:&nbsp;</p>\n\n<ul>\n\t<li>Am I putting others before myself right now?</li>\n\t<li>How can I love the people around me if I&rsquo;m not even paying attention to them?</li>\n\t<li>What if Jesus wants to speak through me, but I&rsquo;ve got my nose stuck in my phone?&nbsp;</li>\n</ul>\n\n<p>The world won&rsquo;t implode if we don&rsquo;t reply to every text or email immediately. And that Instagram picture doesn&rsquo;t need to go up ASAP. If it&rsquo;s already been a week between posts, what&rsquo;s a few more hours? We love everyone better &mdash; our friends to our coworkers to our social media followers &mdash; when we give them the gift of our full attention.&nbsp;</p>\n\n<p>I can almost hear Jesus saying, &ldquo;Greater love has no one than this: to lay down one&rsquo;s phone for one&rsquo;s loved ones&rdquo;&nbsp; (John 15:13).&nbsp;</p>\n\n<h3>Four Ways To Stay Away From Your Phone And Be Intentionally Present</h3>\n\n<p><a href=\'http://newspring.cc/christmas\' target=\'_blank\'>This holiday season</a>, we will have more opportunities to be around friends and family than usual. So let&rsquo;s make the most of each minute. The following are some tips can help us all be a little more present.&nbsp;&nbsp;</p>\n\n<ol>\n\t<li>Turn your phone off, leave it in the car, or put it in airplane mode. Habitual notification checks can take a break.</li>\n\t<li>Talk about something other than yourself. Ask someone else about their life (job, school, kids).</li>\n\t<li>Look for opportunities to encourage someone. Ask the Holy Spirit to speak through you and help you love people.</li>\n\t<li>Ask Jesus to keep your heart interested and engaged in what really matters. He wants to help!</li>\n</ol>\n\n<p>Intentionally taking advantage of time with loved ones sometimes does require sacrificing our own immediate desires or comforts, but that&rsquo;s what we are commanded to do (Philippians 2:3).&nbsp;</p>\n\n<p>Take pictures, but don&rsquo;t let the only memory be a digital one. The positive outcome of interactions should be more than Instagram likes. Unplug from social media, choose to be present, and you will be surprised at the good it does for your heart and the hearts of those around you.&nbsp;</p>'}
        </HTMLView>
      </Providers>,
    );
    expect(tree).toMatchSnapshot();
  });
});
