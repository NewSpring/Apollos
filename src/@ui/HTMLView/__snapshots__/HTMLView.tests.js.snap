// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the HTMLView component renders HTML to React 1`] = `
<View>
  <View
    style={
      Object {
        "paddingVertical": 10,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      bold={false}
      ellipsizeMode="tail"
      italic={false}
      style={
        Object {
          "color": "#303030",
          "fontFamily": "DroidSerif-Regular",
          "fontSize": 20.02,
          "lineHeight": 33.11,
          "paddingBottom": 2.25,
          "paddingTop": 1.8,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
      >
        I sat on my couch next to my high school 
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        onPress={[Function]}
        style={
          Object {
            "color": "#6bac43",
            "textDecorationLine": "underline",
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
        >
          Fuse Group
        </Text>
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
      >
        , silently fuming at their inability to put their phones down during Bible study. Finally, I spoke up, teasing one of the girls in a tone that clearly said, “I’m not actually kidding.” 
      </Text>
    </Text>
  </View>
  <View
    style={
      Object {
        "paddingVertical": 10,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      bold={false}
      ellipsizeMode="tail"
      italic={false}
      style={
        Object {
          "color": "#303030",
          "fontFamily": "DroidSerif-Regular",
          "fontSize": 20.02,
          "lineHeight": 33.11,
          "paddingBottom": 2.25,
          "paddingTop": 1.8,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
      >
        A few days later, I was in the car with my sister, and as I sat in the passenger seat responding to emails, I heard her laugh and say, “You always mumble when you are typing. Stop doing that.” 
      </Text>
    </Text>
  </View>
  <View
    style={
      Object {
        "paddingVertical": 10,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      bold={false}
      ellipsizeMode="tail"
      italic={false}
      style={
        Object {
          "color": "#303030",
          "fontFamily": "DroidSerif-Regular",
          "fontSize": 20.02,
          "lineHeight": 33.11,
          "paddingBottom": 2.25,
          "paddingTop": 1.8,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
      >
        “Sorry,” I responded. “I just don’t want people to think I’m ignoring them when I’m typing.”
      </Text>
    </Text>
  </View>
  <View
    style={
      Object {
        "paddingVertical": 10,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      bold={false}
      ellipsizeMode="tail"
      italic={false}
      style={
        Object {
          "color": "#303030",
          "fontFamily": "DroidSerif-Regular",
          "fontSize": 20.02,
          "lineHeight": 33.11,
          "paddingBottom": 2.25,
          "paddingTop": 1.8,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
      >
        But the truth is, I am ignoring them! In my head, this behavior is totally acceptable. I tell myself I’m tending to my priorities in order, and in doing so, I send a message to the person in front of me that she is of lesser importance.
      </Text>
    </Text>
  </View>
  <View
    style={
      Object {
        "alignSelf": "stretch",
        "borderBottomWidth": 0.5,
        "borderColor": "#858585",
        "borderTopWidth": 0.5,
        "marginVertical": 40,
        "paddingHorizontal": 20,
        "paddingVertical": 30,
      }
    }
  >
    <View
      style={
        Object {
          "paddingVertical": 10,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        bold={false}
        ellipsizeMode="tail"
        italic={false}
        style={
          Object {
            "color": "#303030",
            "fontFamily": "DroidSerif-RegularItalic",
            "fontSize": 20.02,
            "lineHeight": 33.11,
            "paddingBottom": 2.25,
            "paddingTop": 1.8,
            "textAlign": "center",
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
        >
          My phone is the primary outlet that distracts me from people who need my attention.
        </Text>
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "paddingVertical": 10,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      bold={false}
      ellipsizeMode="tail"
      italic={false}
      style={
        Object {
          "color": "#303030",
          "fontFamily": "DroidSerif-Regular",
          "fontSize": 20.02,
          "lineHeight": 33.11,
          "paddingBottom": 2.25,
          "paddingTop": 1.8,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
      >
        I quickly get frustrated when I am the one on the outside, but how often to I shut others out during my own social media binges or email frenzies? Just like my girls, I’m learning that my phone is the primary outlet that distracts me from people who need my attention.
      </Text>
    </Text>
  </View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#303030",
        "fontFamily": "Colfax-Bold",
        "fontSize": 32.4,
        "lineHeight": 41.04,
        "paddingBottom": 2.7,
        "paddingTop": 5.4,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
      >
        Why Being Present Matters
      </Text>
    </Text>
  </Text>
  <View
    style={
      Object {
        "paddingVertical": 10,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      bold={false}
      ellipsizeMode="tail"
      italic={false}
      style={
        Object {
          "color": "#303030",
          "fontFamily": "DroidSerif-Regular",
          "fontSize": 20.02,
          "lineHeight": 33.11,
          "paddingBottom": 2.25,
          "paddingTop": 1.8,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
      >
        There is nothing wrong with using our phones — as long as we remember that the best investment we can make is to invest in our relationships with Jesus and each other. We were created for 
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        onPress={[Function]}
        style={
          Object {
            "color": "#6bac43",
            "textDecorationLine": "underline",
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
        >
          community
        </Text>
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
      >
         by God, who exists in community as Father, Son, and Holy Spirit (John 1:1-2, Genesis 2:18). 
      </Text>
    </Text>
  </View>
  <View
    style={
      Object {
        "paddingVertical": 10,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      bold={false}
      ellipsizeMode="tail"
      italic={false}
      style={
        Object {
          "color": "#303030",
          "fontFamily": "DroidSerif-Regular",
          "fontSize": 20.02,
          "lineHeight": 33.11,
          "paddingBottom": 2.25,
          "paddingTop": 1.8,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
      >
        Our phones help us stay connected, but they are not the foundation of all connection. If we’re honest, our phones are much quicker to distract and disengage us from the people right in front of us. 
      </Text>
    </Text>
  </View>
  <View
    style={
      Object {
        "paddingVertical": 10,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      bold={false}
      ellipsizeMode="tail"
      italic={false}
      style={
        Object {
          "color": "#303030",
          "fontFamily": "DroidSerif-Regular",
          "fontSize": 20.02,
          "lineHeight": 33.11,
          "paddingBottom": 2.25,
          "paddingTop": 1.8,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
      >
        Jesus said loving God and loving others is greatest of all God’s commandments (Matthew 22:38-40). And, after trying everything the world had to offer, King Solomon reminded his people that their relationships with God and others are the only things that really matter (Ecclesiastes 2:24-26, Ecclesiastes 3:9). 
      </Text>
    </Text>
  </View>
  <View
    style={
      Object {
        "alignSelf": "stretch",
        "borderBottomWidth": 0.5,
        "borderColor": "#858585",
        "borderTopWidth": 0.5,
        "marginVertical": 40,
        "paddingHorizontal": 20,
        "paddingVertical": 30,
      }
    }
  >
    <View
      style={
        Object {
          "paddingVertical": 10,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        bold={false}
        ellipsizeMode="tail"
        italic={false}
        style={
          Object {
            "color": "#303030",
            "fontFamily": "DroidSerif-RegularItalic",
            "fontSize": 20.02,
            "lineHeight": 33.11,
            "paddingBottom": 2.25,
            "paddingTop": 1.8,
            "textAlign": "center",
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
        >
          Greater love has no one than this: to lay down one’s phone for one’s loved ones.
        </Text>
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "paddingVertical": 10,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      bold={false}
      ellipsizeMode="tail"
      italic={false}
      style={
        Object {
          "color": "#303030",
          "fontFamily": "DroidSerif-Regular",
          "fontSize": 20.02,
          "lineHeight": 33.11,
          "paddingBottom": 2.25,
          "paddingTop": 1.8,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
      >
        So the next time we feel like we have to text back immediately or tweet before someone else does, consider: 
      </Text>
    </Text>
  </View>
  <View
    style={
      Object {
        "flexDirection": "row",
      }
    }
  >
    <View
      style={
        Object {
          "paddingRight": 9,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        bold={false}
        ellipsizeMode="tail"
        italic={false}
        style={
          Object {
            "color": "#303030",
            "fontFamily": "DroidSerif-Regular",
            "fontSize": 20.02,
            "lineHeight": 33.11,
            "paddingBottom": 2.25,
            "paddingTop": 1.8,
          }
        }
      >
        •
      </Text>
    </View>
    <View
      style={
        Object {
          "flexShrink": 1,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        bold={false}
        ellipsizeMode="tail"
        italic={false}
        style={
          Object {
            "color": "#303030",
            "fontFamily": "DroidSerif-Regular",
            "fontSize": 20.02,
            "lineHeight": 33.11,
            "paddingBottom": 2.25,
            "paddingTop": 1.8,
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
        >
          Am I putting others before myself right now?
        </Text>
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "flexDirection": "row",
      }
    }
  >
    <View
      style={
        Object {
          "paddingRight": 9,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        bold={false}
        ellipsizeMode="tail"
        italic={false}
        style={
          Object {
            "color": "#303030",
            "fontFamily": "DroidSerif-Regular",
            "fontSize": 20.02,
            "lineHeight": 33.11,
            "paddingBottom": 2.25,
            "paddingTop": 1.8,
          }
        }
      >
        •
      </Text>
    </View>
    <View
      style={
        Object {
          "flexShrink": 1,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        bold={false}
        ellipsizeMode="tail"
        italic={false}
        style={
          Object {
            "color": "#303030",
            "fontFamily": "DroidSerif-Regular",
            "fontSize": 20.02,
            "lineHeight": 33.11,
            "paddingBottom": 2.25,
            "paddingTop": 1.8,
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
        >
          How can I love the people around me if I’m not even paying attention to them?
        </Text>
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "flexDirection": "row",
      }
    }
  >
    <View
      style={
        Object {
          "paddingRight": 9,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        bold={false}
        ellipsizeMode="tail"
        italic={false}
        style={
          Object {
            "color": "#303030",
            "fontFamily": "DroidSerif-Regular",
            "fontSize": 20.02,
            "lineHeight": 33.11,
            "paddingBottom": 2.25,
            "paddingTop": 1.8,
          }
        }
      >
        •
      </Text>
    </View>
    <View
      style={
        Object {
          "flexShrink": 1,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        bold={false}
        ellipsizeMode="tail"
        italic={false}
        style={
          Object {
            "color": "#303030",
            "fontFamily": "DroidSerif-Regular",
            "fontSize": 20.02,
            "lineHeight": 33.11,
            "paddingBottom": 2.25,
            "paddingTop": 1.8,
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
        >
          What if Jesus wants to speak through me, but I’ve got my nose stuck in my phone? 
        </Text>
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "paddingVertical": 10,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      bold={false}
      ellipsizeMode="tail"
      italic={false}
      style={
        Object {
          "color": "#303030",
          "fontFamily": "DroidSerif-Regular",
          "fontSize": 20.02,
          "lineHeight": 33.11,
          "paddingBottom": 2.25,
          "paddingTop": 1.8,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
      >
        The world won’t implode if we don’t reply to every text or email immediately. And that Instagram picture doesn’t need to go up ASAP. If it’s already been a week between posts, what’s a few more hours? We love everyone better — our friends to our coworkers to our social media followers — when we give them the gift of our full attention. 
      </Text>
    </Text>
  </View>
  <View
    style={
      Object {
        "paddingVertical": 10,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      bold={false}
      ellipsizeMode="tail"
      italic={false}
      style={
        Object {
          "color": "#303030",
          "fontFamily": "DroidSerif-Regular",
          "fontSize": 20.02,
          "lineHeight": 33.11,
          "paddingBottom": 2.25,
          "paddingTop": 1.8,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
      >
        I can almost hear Jesus saying, “Greater love has no one than this: to lay down one’s phone for one’s loved ones”  (John 15:13). 
      </Text>
    </Text>
  </View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#303030",
        "fontFamily": "Colfax-Bold",
        "fontSize": 32.4,
        "lineHeight": 41.04,
        "paddingBottom": 2.7,
        "paddingTop": 5.4,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
      >
        Four Ways To Stay Away From Your Phone And Be Intentionally Present
      </Text>
    </Text>
  </Text>
  <View
    style={
      Object {
        "paddingVertical": 10,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      bold={false}
      ellipsizeMode="tail"
      italic={false}
      style={
        Object {
          "color": "#303030",
          "fontFamily": "DroidSerif-Regular",
          "fontSize": 20.02,
          "lineHeight": 33.11,
          "paddingBottom": 2.25,
          "paddingTop": 1.8,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        onPress={[Function]}
        style={
          Object {
            "color": "#6bac43",
            "textDecorationLine": "underline",
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
        >
          This holiday season
        </Text>
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
      >
        , we will have more opportunities to be around friends and family than usual. So let’s make the most of each minute. The following are some tips can help us all be a little more present.  
      </Text>
    </Text>
  </View>
  <View
    style={
      Object {
        "flexDirection": "row",
      }
    }
  >
    <View
      style={
        Object {
          "paddingRight": 9,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        bold={false}
        ellipsizeMode="tail"
        italic={false}
        style={
          Object {
            "color": "#303030",
            "fontFamily": "DroidSerif-Regular",
            "fontSize": 20.02,
            "lineHeight": 33.11,
            "paddingBottom": 2.25,
            "paddingTop": 1.8,
          }
        }
      >
        •
      </Text>
    </View>
    <View
      style={
        Object {
          "flexShrink": 1,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        bold={false}
        ellipsizeMode="tail"
        italic={false}
        style={
          Object {
            "color": "#303030",
            "fontFamily": "DroidSerif-Regular",
            "fontSize": 20.02,
            "lineHeight": 33.11,
            "paddingBottom": 2.25,
            "paddingTop": 1.8,
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
        >
          Turn your phone off, leave it in the car, or put it in airplane mode. Habitual notification checks can take a break.
        </Text>
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "flexDirection": "row",
      }
    }
  >
    <View
      style={
        Object {
          "paddingRight": 9,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        bold={false}
        ellipsizeMode="tail"
        italic={false}
        style={
          Object {
            "color": "#303030",
            "fontFamily": "DroidSerif-Regular",
            "fontSize": 20.02,
            "lineHeight": 33.11,
            "paddingBottom": 2.25,
            "paddingTop": 1.8,
          }
        }
      >
        •
      </Text>
    </View>
    <View
      style={
        Object {
          "flexShrink": 1,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        bold={false}
        ellipsizeMode="tail"
        italic={false}
        style={
          Object {
            "color": "#303030",
            "fontFamily": "DroidSerif-Regular",
            "fontSize": 20.02,
            "lineHeight": 33.11,
            "paddingBottom": 2.25,
            "paddingTop": 1.8,
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
        >
          Talk about something other than yourself. Ask someone else about their life (job, school, kids).
        </Text>
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "flexDirection": "row",
      }
    }
  >
    <View
      style={
        Object {
          "paddingRight": 9,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        bold={false}
        ellipsizeMode="tail"
        italic={false}
        style={
          Object {
            "color": "#303030",
            "fontFamily": "DroidSerif-Regular",
            "fontSize": 20.02,
            "lineHeight": 33.11,
            "paddingBottom": 2.25,
            "paddingTop": 1.8,
          }
        }
      >
        •
      </Text>
    </View>
    <View
      style={
        Object {
          "flexShrink": 1,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        bold={false}
        ellipsizeMode="tail"
        italic={false}
        style={
          Object {
            "color": "#303030",
            "fontFamily": "DroidSerif-Regular",
            "fontSize": 20.02,
            "lineHeight": 33.11,
            "paddingBottom": 2.25,
            "paddingTop": 1.8,
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
        >
          Look for opportunities to encourage someone. Ask the Holy Spirit to speak through you and help you love people.
        </Text>
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "flexDirection": "row",
      }
    }
  >
    <View
      style={
        Object {
          "paddingRight": 9,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        bold={false}
        ellipsizeMode="tail"
        italic={false}
        style={
          Object {
            "color": "#303030",
            "fontFamily": "DroidSerif-Regular",
            "fontSize": 20.02,
            "lineHeight": 33.11,
            "paddingBottom": 2.25,
            "paddingTop": 1.8,
          }
        }
      >
        •
      </Text>
    </View>
    <View
      style={
        Object {
          "flexShrink": 1,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        bold={false}
        ellipsizeMode="tail"
        italic={false}
        style={
          Object {
            "color": "#303030",
            "fontFamily": "DroidSerif-Regular",
            "fontSize": 20.02,
            "lineHeight": 33.11,
            "paddingBottom": 2.25,
            "paddingTop": 1.8,
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
        >
          Ask Jesus to keep your heart interested and engaged in what really matters. He wants to help!
        </Text>
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "paddingVertical": 10,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      bold={false}
      ellipsizeMode="tail"
      italic={false}
      style={
        Object {
          "color": "#303030",
          "fontFamily": "DroidSerif-Regular",
          "fontSize": 20.02,
          "lineHeight": 33.11,
          "paddingBottom": 2.25,
          "paddingTop": 1.8,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
      >
        Intentionally taking advantage of time with loved ones sometimes does require sacrificing our own immediate desires or comforts, but that’s what we are commanded to do (Philippians 2:3). 
      </Text>
    </Text>
  </View>
  <View
    style={
      Object {
        "paddingVertical": 10,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      bold={false}
      ellipsizeMode="tail"
      italic={false}
      style={
        Object {
          "color": "#303030",
          "fontFamily": "DroidSerif-Regular",
          "fontSize": 20.02,
          "lineHeight": 33.11,
          "paddingBottom": 2.25,
          "paddingTop": 1.8,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
      >
        Take pictures, but don’t let the only memory be a digital one. The positive outcome of interactions should be more than Instagram likes. Unplug from social media, choose to be present, and you will be surprised at the good it does for your heart and the hearts of those around you. 
      </Text>
    </Text>
  </View>
</View>
`;

exports[`the HTMLView component strips out badly formatted links 1`] = `
<View>
  <View
    style={
      Object {
        "paddingVertical": 10,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      bold={false}
      ellipsizeMode="tail"
      italic={false}
      style={
        Object {
          "color": "#303030",
          "fontFamily": "DroidSerif-Regular",
          "fontSize": 20.02,
          "lineHeight": 33.11,
          "paddingBottom": 2.25,
          "paddingTop": 1.8,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
      >
        Bad link
      </Text>
    </Text>
  </View>
</View>
`;
